clientSide/theteddyproject/src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,116 @@
+.navbar
+{ 
+    height: 60px;
+    width: 100%;
+    display: flex;
+    background-color:#D36B00;
+}
+.color
+{
+    height: 200px;
+    width: 200px;
+    margin: 20px; 
+    border-radius: 10%;  
+}
+.toys
+{
+    margin-left: 50px;
+    margin-top: 20px;
+    color: white;
+}
+.login
+{
+    height:21px;
```

Comment:
Please mainitain one class name for common properties

Reply:
okay i will use common propertis